build.sbt

```diff
@@ -62,10 +62,10 @@ val relocationPrefix = s"$myPackage.repackaged"
 
 libraryDependencies ++= Seq(
   // Keep com.google.cloud dependencies in sync
-  "com.google.cloud" % "google-cloud-bigquery" % "1.96.0",
-  "com.google.cloud" % "google-cloud-bigquerystorage" % "0.114.0-beta",
+  "com.google.cloud" % "google-cloud-bigquery" % "1.99.0",
+  "com.google.cloud" % "google-cloud-bigquerystorage" % "0.117.0-beta",
   // Keep in sync with com.google.cloud
```

Comment:
How should these be kept in sync? If there's a table somewhere, then can you add a link?

Reply:
Not sure if there is a table

I use sbt dependencyTree personally, but you can link to github at the release commit

Reply:
The best place is the google-cloud-java BOM (cc) but unfortunately
a. com.google.cloud:google-cloud-bigquerystorage is missing (I've created https://github.com/googleapis/google-cloud-java/issues/6710)
b. SBT does not support BOM yet (See https://github.com/sbt/sbt/issues/4531)

Perhaps we can move to maven/gradle at some point, especially as the DataSources v2 API is java based rather than scala.

The BOM can be still used as a reference for the versions